rebasehelper/application.py

```diff
@@ -768,43 +790,38 @@ def run(self):
                     build = self.build_packages()
                     if self.conf.builds_nowait and not self.conf.build_tasks:
                         return
-                except RuntimeError:
-                    logger.error('Unknown error caused by build log analysis')
-                    # TODO: exception should be raised instead of returning a value - it is never checked!
```

Comment:
The comment was there for a reason - return value of `Application.run` is just ignored, so it would be good to do something about it. Either handle the return value in `ClieHelper.run`, on reraise the exception.